Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -454,8 +477,99 @@ class CredentialsManagerSpec: QuickSpec {
                         }
                     }
                 }
+
             }
-            
+
+            context("forced renew") {
+
+                it("should not yield a new access token by default") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == AccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should not yield a new access token without a new scope") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn), scope: "openid profile")
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: nil, minTTL: 0) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == AccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should not yield a new access token with the same scope") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn), scope: "openid profile")
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: "openid profile", minTTL: 0) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == AccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should yield a new access token with a new scope") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn), scope: "openid profile")
```

Comment:
similar to the comment above, swap the scope values.

Reply:
Thanks, changed in https://github.com/auth0/Auth0.swift/pull/399/commits/548ee4472061e2ac1974c36320b6ba2b4004f2fc.